plugins/services/services_inspector.rb

```diff
@@ -35,10 +35,17 @@ def inspect(_filter, _options = {})
         init_system: "systemd"
       )
     elsif @system.has_command?("initctl")
-      result = ServicesScope.new(
-        inspect_ubuntu_services,
-        init_system: "upstart"
-      )
+      if @system.has_command?("chkconfig")
```

Comment:
No, as then for RHEL6 still sysvinit would be the recorded init system, which is wrong.